src/workerd/io/actor-cache.h

```diff
@@ -179,6 +179,10 @@ class ActorCache final: public ActorCacheInterface {
   class SharedLru;
   // Shared LRU for a whole isolate.
 
+  static constexpr auto SHUTDOWN_ERROR_MESSAGE =
+      "broken.ignored; jsg.Error: "
+      "Durable Object is shutting down, storage is no longer accessible."_kj;
```

Comment:
I think it'll be easier for me see on the `ew-test` side of things, but a user wouldn't actually receive _this_ error if they broke their Actor via OOM/CPU-usage/throw-from-blockConcurrencyWhile, or if something like a storage error happened, right? I assume they'd receive the real underlying error tunneled to their eyeball worker? Otherwise I can see this being very confusing. ("Why is my Actor shutting down? What does shutting down even mean?")

Actually, maybe what I need to better understand is when they _would_ see this error. I imagined an active Actor would only be shutdown because it was broken, implying some other exception was thrown.

Reply:
> Actually, maybe what I need to better understand is when they would see this error. I imagined an active Actor would only be shutdown because it was broken, implying some other exception was thrown.

Yeah, in practice this is only a consequence of marking an Actor as broken. That is the current state of the world internally without any code changes.

> a user wouldn't actually receive this error if they broke their Actor via ...

Most notably, code update comes to mind as a situation where the actor is broken but JS could theoretically keep running. We want to have this case covered since we don't want to allow any future operations to work. If we had the error from the brokenness in hand, we could use that instead. That's doable, we'd just need to modify `Worker::Actor::shutdown()` to take an additional argument.

Reply:
> Most notably, code update comes to mind

Ah yeah. I get that we need a fallback error, I just wanted to be sure that they'd get the OOM/CPU/etc error if that were the root cause, and it wasn't obvious to me that that was how it worked.

Reply:
> Most notably, code update comes to mind as a situation where the actor is broken but JS could theoretically keep running

Thanks for calling that out. This might be slightly disruptive then in that cached reads will no longer succeed during a code upgrade, which I wouldn't be surprised if some users noticed. Not that that's a bad thing, per se -- it's nice to rule out the possibility of stale reads.